pkg/reconciler/brokerstandalone/resources/dispatcher.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"fmt"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"knative.dev/eventing/pkg/apis/eventing"
+	eventingv1 "knative.dev/eventing/pkg/apis/eventing/v1"
+
+	"knative.dev/pkg/apis"
+	"knative.dev/pkg/kmeta"
+	"knative.dev/pkg/system"
+)
+
+const (
+	dispatcherContainerName = "dispatcher"
+)
+
+// DispatcherArgs are the arguments to create a Broker's Dispatcher Deployment that handles
+// DeadLetterSink deliveries.
+type DispatcherArgs struct {
+	Broker *eventingv1.Broker
+	Image  string
+	//ServiceAccountName string
+	RabbitMQHost       string
+	RabbitMQSecretName string
+	QueueName          string
+	BrokerUrlSecretKey string
+	BrokerIngressURL   *apis.URL
+	Subscriber         *apis.URL
+}
+
+func DispatcherName(brokerName string) string {
+	return fmt.Sprintf("%s-dlq-dispatcher", brokerName)
```

Comment:
should this have something with the standalone name? maybe it is fine because you can't have two brokers of the same kind in the cluster of standalone vs not... ok I think it is ok